model-archiver/README.md

```diff
@@ -138,12 +138,14 @@ A serialized file (.pt or .pth) should be a checkpoint in case of torchscript an
 
 ### Handler
 
-Handler can be TorchServe's inbuilt handler name or path to a py file to handle custom TorchServe inference logic. TorchServe supports following handlers out or box:
+Handler can be TorchServe's inbuilt handler name or path to a py file to handle custom TorchServe inference logic. TorchServe supports the following handlers out of box:
 1. `image_classifier`
 2. `object_detector`
 3. `text_classifier`
 4. `image_segmenter`
 
+For a more comprehensive list of built in handlers, make sure to checkout the [examples](../model-archiver/README.md)
```

Comment:
@msaroufim seems this link loops back to the same page, is it meant to be https://github.com/pytorch/serve/blob/master/docs/default_handlers.md?